autogpt_platform/backend/backend/blocks/allquiet/triggers_test.py

```
@@ -0,0 +1,149 @@
+            json.loads(_render(RECOMMENDED_BODY_TEMPLATE, attributes=[]))["attributes"]
+            == []
+        )
+
+
+def _render(template: str, *, attributes: list[dict]) -> str:
+    """Minimal Handlebars stand-in for the subset the template uses.
+
+    Only `{{#each}}` / `{{#unless @last}}` / `{{field}}` are needed, so this
+    avoids adding a Handlebars dependency just to assert the template is valid.
+    """
+    each = re.search(r"\{\{#each attributes\}\}(.*?)\{\{/each\}\}", template, re.S)
```

Ruff (0.16.1)

[error] Use of regular expression alias `re.S`

Replace with `re.DOTALL`

(FURB167)